tests/console/validate_autoyast.pm

```diff
@@ -0,0 +1,50 @@
+# SUSE's openQA tests
+#
+# Copyright © 2019 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved.  This file is offered as-is,
+# without any warranty.
+
+# Summary: smoke test for autoyast post-installation
+# Maintainer: Yiannis Bonatakis <ybonatakis@suse.com>
+
+use strict;
+use warnings;
+use base "opensusebasetest";
+use testapi;
+use utils;
+use y2logsstep;
+use version_utils 'is_opensuse';
+
+sub run {
+    my $self = shift;
+    $self->select_serial_terminal;
+
+    record_info('INFO', 'Check environment');
+    assert_script_run "env | grep \"SHELL=/bin/bash\"";
+    assert_script_run "env | grep \"HOME=\/root\"";
+    assert_script_run "env | grep \"OSTYPE=linux\"";
+
+    record_info('INFO', 'Check user');
+    assert_script_run "id -u $username |grep 1000";
+
+    my $service = opensusebasetest::firewall();
+    if (not is_opensuse()) ? record_info('INFO', 'Check firewall is enabled and running') :
```

Comment:
Can you state why you think it's depending on openSUSE/SLE? I think it's rather depending on the versions within each distribution

Reply:
the initial condition was intended to distinguish sle from opensuse based on the DISTRI variable. then i become award of the use of is_opensuse function from one of the comments. And seems it does the same thing. In the xml for opensuse there is not firewall configuration. But i havent checked any version to make sure if something change